framework/wazuh/core/indexer/bulk.py

```diff
@@ -0,0 +1,199 @@
+from typing import Optional, Dict, Any, List, Protocol, Coroutine
+from enum import Enum
+
+from opensearchpy import AsyncOpenSearch
+
+
+class BulkAction(str, Enum):
+    """Enum representing bulk actions for OpenSearch.
```

Comment:
```suggestion
    """Enum representing bulk actions for the Indexer.
```

Reply:
Changed in f3067e1c80a1254db438f9ef7b4bc8535cd89e81.